loom/graphics/gfxQuadRenderer.cpp

```diff
@@ -106,29 +90,18 @@ void QuadRenderer::submit()
 
             if (!Graphics_IsGLStateValid(GFX_OPENGL_STATE_QUAD))
             {
-                // Select the shader.
-                if(sTinted)

```

Comment:
We haven't reimplemented the non-tint version since the GL switch yet, so that's why the shaders were the same, should this be reconsidered?